README.md

```diff
@@ -35,30 +35,41 @@ Screenshot (will need to be updated as syntax coloring and UI polish is added):
 
 ![xi screenshot](/doc/img/xi-mac-screenshot.png?raw=true)
 
-
 ## Getting started
 
 ### Requirements
+
 - [Xcode 9.x](https://developer.apple.com/xcode/)
 - [Rust](https://www.rust-lang.org/). We test against the latest stable version,
-and recommend installing through [rustup](https://rustup.rs).
+and recommend installing through [rustup](https://rustup.rs)
```

Comment:
while i'm nitting, I think this is maybe clearer with the full stop/period?

Reply:
Sorry, I must have missed that when I was coping and pasting from master.